Person/persons.md

```diff
@@ -0,0 +1,386 @@
+*endDate* | `String` | end date of living by the person
+*lattitude* | `String` | lattitude of the location of the person
+*longitude* | `String` | longitude of the location of the person
+
+
+```console
+POST person/:target_person_uuid/address
+{ 
+    "address1": "30, Vivekananda Layout, Munnekolal,Marathahalli",
+    "cityVillage": "Bengaluru"
+    "stateProvince": "Karnataka",
+    "postalCode": "560037",
+    "lattitude": "28.65033",
+    "longitude": "77.304255"
+}
+```
+
+### Update person address subresource
+
+* To update a person address with given uuid value for a specific person resource you need to specify below properties in your request body.
+If user not logged in to perform this action, a `401 Unauthorized` status returned.
+
+#### Attributes
+
+Parameter | Type | Description
+--- | --- | ---
+*preferred* | `Boolean` | preferred/not preferred address
+*address1* | `String` | address of the person
+*address2* | `String` | second address of the person
+*cityVillage* | `String` | city/village of the person
+*stateProvince* | `String` | name of the state of the person
+*country* | `String` | country of the person
+*postalCode* | `String` | pin code of the person
+*countyDistrict* | `String` | county district of the person
+*address3* | `String` | third address of the person
+*address4* | `String` | fourth address of the person
+*address5* | `String` | fifth address of the person
+*address6* | `String` | sixth address of the person
+*startDate* | `String` | start date of living by the person
+*endDate* | `String` | end date of living by the person
+*lattitude* | `String` | lattitude of the location of the person
+*longitude* | `String` | longitude of the location of the person
+
+
+```console
+POST person/:target_person_uuid/address
+{ 
+    "address1": "30, Vivekananda Layout, Munnekolal,Marathahalli",
+    "cityVillage": "Bengaluru"
+    "stateProvince": "Karnataka",
+    "postalCode": "560037",
+    "lattitude": "28.65033",
+    "longitude": "77.304255"
+}
+```
+
+### Delete a person address sub resource
+
+* Delete or retire a target address sub resource by its UUID.Returns a `404 Not Found` status if attribute not exists. 
+If user not logged in to perform this action, a `401 Unauthorized` status returned.
+
+ ```console
+DELETE /person/:target_person_uuid/person/:target_address_uuid
+ ```
+
+### List person attribute subresource
+
+* List all the person attribute subresource corresponding to a `target_person_uuid`. Returns a `404 Not Found` status if person attribute not exists. If user not logged in to perform this action, a `401 unauthorized` status returned.
+
+```console
+GET /person/:target_person_uuid/attribute
+```
+
+* List all the person attribute subresource by it's `UUID` and corresponding to a `target_person_uuid`. Returns a `404 Not Found` status if person attribute not exists. If user not logged in to perform this action, a `401 unauthorized` status returned.
```

Comment:
* List all person attributes for a given person. Returns a `404 Not Found` if the person doesn't exist. If not authenticated or the authenticated user does not have sufficient privileges, a `401 Unauthorized` status is returned.